docker-compose.yml

```diff
@@ -1,17 +1,48 @@
-version: '3'
+version: "3"
 services:
   exp:
-    build: .
+    build:
+      context: .
+      args:
+        build_date: ""
+        build_base_image: $BASE_IMAGE
+        build_distro_version: $XP_VERSION
     restart: always
+    deploy:
+      resources:
+        limits:
+          cpus: "2.00"
+          memory: "1024M"
     ports:
+      - "1099:1099"
+      - "2609:2609"
+      - "3000:3000" # Profiling test
+      - "3001:3001" # Profiling test
+      - "4848:4848"
+      - "5005:5005" # Remote debugging test
```

Comment:
3000 3001 and 5005 look open just for testing. Is this file for testing only?

Reply:
Yes the docker-compose file is only for local testing while developing the image. This is not the default behavior when it is used.